content/blog/highly-available-nfs-workload-on-hpe-greenlake-for-private-cloud-enterprise-using-serviceguard-for-linux.md

```diff
@@ -0,0 +1,748 @@
+         asset_tag        = "vm_terraform_sglx_quorum"
+         folder_code      = data.hpegl_vmaas_cloud_folder.compute_folder.code
+         create_user      = true
+     }
+   }
+```
+
+3 VMs need to be created to setup SGLX. 2 VMs will be used to create Serviceguard for Linux nodes where the NFS service will be up and running. The third VM will act as a quorum server for the Serviceguard cluster to ensure that split brain of the cluster does not impact the availability of the monitored workload.
+
+**Note:** You can get information about each of the resource statements supported by the hpegl provider from [GitHub](https://github.com/hpe/terraform-provider-hpegl/tree/main/docs/resources).
+
+**Note:** An existing Serviceguard Quorum Server in your environment can be used instead of provisioning a third VM, provided the Quorum Server is reachable to the 2 VM’s that were created.
+
+### Terraform init
+
+Before you can use Terraform, you need to initialize it from the configuration file we have created. This is done with the following step: 
+
+`terraform init`
+
+### Terraform ready to plan
+
+To validate your configuration file, it is recommended to run the terraform validate command. Once ready, the terraform plan command will provide the a summary of the deployment that would be built when terraform apply method would be  used.
+Once you agree with the plan and confirm, you can apply the configuration.
+
+### Terraform ready to apply
+
+The command you need to use is now: 
+
+```shell
+terraform apply
+```
+
+ This will rerun the plan command, then prompt you to confirm before it starts building what is in the plan:
+Here is some sample output from the terraform apply command:
+
+```
+Do you want to perform these actions?
+  Terraform will perform the actions described above.
+  Only 'yes' will be accepted to approve.
+
+  Enter a value: yes
+
+hpegl_vmaas_instance.my_drbd\[1]: Creating...
+hpegl_vmaas_instance.my_quorum\[0]: Creating...
+hpegl_vmaas_instance.my_drbd\[0]: Creating...
+hpegl_vmaas_instance.my_quorum\[0]: Still creating... \[10s elapsed]
+hpegl_vmaas_instance.my_drbd\[1]: Still creating... \[10s elapsed]
+hpegl_vmaas_instance.my_drbd\[1]: Still creating... \[10s elapsed]
+
+hpegl_vmaas_instance.my_drbd\[1]: Creation complete after 2m8s \[id=3105]
+hpegl_vmaas_instance.my_drbd\[0]: Creation complete after 2m8s \[id=3111]
+hpegl_vmaas_instance.my_quorum\[0]: Creation complete after 2m8s \[id=3108]
+
+Apply complete! Resources: 4 added, 0 changed, 0 destroyed.
+```
+
+Once the command completes, your virtual machines are ready.
+
+# Configuring a Highly Available NFS solution
+
+Now that the VMs are provisioned, we can now deploy HPE Serviceguard for Linux on these VMs to create a cluster to provide high availability for the applications running on the VMs, NFS server in this case.
+
+## Installing Serviceguard for Linux
+
+Serviceguard and all its components can be installed using Ansible playbooks 
```

Comment:
Suggestion for changes:

Serviceguard and all its components can be installed using Ansible playbooks. Clone the repository on Ansible control node: